src/middlewared/middlewared/etc_files/local/openldap/ldap.conf

```diff
@@ -22,11 +22,13 @@
 
         ad = middleware.call_sync('activedirectory.config')
         ad_enabled = ad['enable']
+        idmap = middleware.call_sync('idmap.query',
+                                     [('name', '=', 'DS_TYPE_ACTIVEDIRECTORY')],
+                                     {'get': True})
 
-        if ad['enable'] and ad['idmap_backend'] in ["rfc2307", "ldap"]:
+        if ad['enable'] and idmap in ["rfc2307", "ldap"]:
```

Comment:
Is `idmap` going to be a string at this point ?

Reply:
Heh. This means I've been sitting at the computer too long.